### PR TITLE
feat: Added support for AVIO custom logger components in linter

### PR DESCRIPTION
**Pull request by dileepkumarv-avio**
Implemented support for identifying AVIO Custom logger components as logger components in linter. 
And, updated linter rules and test case to validate AVIO Custom logger components.

### CODE REVIEW

File: mule-linter-core/src/test/groovy/com/avioconsulting/mule/linter/rule/configuration/ConsecutiveLoggersCountRuleTest.groovy [hunk 1]
```diff
@@ -46,8 +48,10 @@ class ConsecutiveLoggersCountRuleTest extends Specification{
         List<RuleViolation> violations = rule.execute(app)
 
         then:
-        violations.size() == 1
+        println violations
```
Comment (neckron) [inline mule-linter-core/src/test/groovy/com/avioconsulting/mule/linter/rule/configuration/ConsecutiveLoggersCountRuleTest.groovy:51]: it's needed to print here?

File: mule-linter-core/src/test/groovy/com/avioconsulting/mule/linter/rule/configuration/ConsecutiveLoggersCountRuleTest.groovy [hunk 2]
```diff
@@ -29,10 +29,12 @@ class ConsecutiveLoggersCountRuleTest extends Specification{
         List<RuleViolation> violations = rule.execute(app)
 
         then:
-        violations.size() == 2
+        println violations
```
Comment (neckron) [inline mule-linter-core/src/test/groovy/com/avioconsulting/mule/linter/rule/configuration/ConsecutiveLoggersCountRuleTest.groovy:32]: it's needed to print here?

File: mule-linter-core/src/test/groovy/com/avioconsulting/mule/linter/rule/configuration/ConsecutiveLoggersCountRuleTest.groovy [hunk 3]
```diff
@@ -66,10 +70,15 @@ class ConsecutiveLoggersCountRuleTest extends Specification{
         List<RuleViolation> violations = rule.execute(app)
 
         then:
+        println violations
```
Comment (neckron) [inline mule-linter-core/src/test/groovy/com/avioconsulting/mule/linter/rule/configuration/ConsecutiveLoggersCountRuleTest.groovy:73]: it's needed to print here?

File: mule-linter-core/src/test/groovy/com/avioconsulting/mule/linter/rule/configuration/ConsecutiveLoggersCountRuleTest.groovy (hunk 2 above)
Comment (dileepkumarv-avio) [reply mule-linter-core/src/test/groovy/com/avioconsulting/mule/linter/rule/configuration/ConsecutiveLoggersCountRuleTest.groovy:32]: Oh, I added that for my testing, will remove this print statement.

**Review by neckron — APPROVED** [review]

**Review by adesjardin — APPROVED** [review]

**Pull request merged**